src/SelfServe/SqlX/SqlX.rp.ts

```diff
@@ -128,3 +128,105 @@ export const refreshDedicatedGatewayProvisioning = async (): Promise<RefreshResu
     return { isUpdateInProgress: false, updateInProgressMessageTKey: undefined };
   }
 };
+
+// Data model for Read Regions Query
```

Comment:
Remove comments here and everywhere else. I d say add comments only when doing something outside the norm, or if it a todo